airflow/kubernetes/pod_generator.py

```diff
@@ -36,6 +36,9 @@
 import kubernetes.client.models as k8s
 import yaml
 from kubernetes.client.api_client import ApiClient
+from ..contrib.kubernetes.pod import (
```

Comment:
Why not use full path?

Reply:
@turbaszek it seems you need to use relative paths if you want to use private functions

Reply:
Hm, I checked out this code in both cases `..` and `airflow.contrib` I see warning `Access to a protected member _extract_volume_mounts of a module`. So I don't think this is a problem.

Reply:
Fixed